package.json

```
@@ -2,6 +2,18 @@
   "name": "ove",
   "private": true,
   "license": "MIT",
+  "scripts": {
+    "install": "lerna bootstrap --hoist",
+    "install:prod": "lerna bootstrap --hoist -- --production --no-optional",
+    "clean": "lerna run clean",
+    "build": "lerna run build",
+    "build:clean": "lerna run clean && lerna run build",
+    "build:docker": "lerna run clean && lerna run build && docker-compose build",
```

Why can't this line be written as:
```suggestion
    "build:docker": "lerna run build:clean && docker-compose build",
```
Worthwhile checking whether other similar optimisations are possible in here.

---

At the moment this is not going to work with the current lerna/npm setup. The lerna commands are going to be executed on the packages which do not have build:clean commands.

---

Ok, I see what you mean. The lerna commands run on the packages and then NPM runs on this top-level package.json. So, there is really no possibility to fix this in that case.